14-05-2015/reverse-polish/reverse-polish-hiren.txt

```diff
@@ -0,0 +1,19 @@
+class ReversePolish
+  def self.evaluate( rp )
+    unless rp.empty?
+      array = rp.split(' ')
```

Comment:
Name the variables meaningfully. `array` for a variable name is too generalized.